addons/core/translations/resources/en-us.yaml

```diff
@@ -465,6 +465,7 @@ worker:
     session_count: Session Count
     ip_address: IP Address
     release_version: Release Version
+    tag_count: '{tagCount, plural, =1 {# tag} other {# tags}}'
```

Comment:
@ZedLi 
WoW I had no idea we could something like this.

NOT BLOCKING: Not sure if we want this logic here, I always understood translation files as plain text. Would you mind to bring this to FE tech time?

Reply:
I can add a topic to tech time but I think this makes the most sense here. This is meant to handle localization differences as some languages have different rules for how pluralization work (in the same way you might have it format different dates).

Reply:
Sure!!

100% Agreed on the differences between languages.

Mentioned to bring it to tech time because this is an interesting topic 😃